install/gcp/installer/forseti_server_installer.py

```diff
@@ -145,17 +153,15 @@ def create_firewall_rules(self):
             self.config.vpc_host_network,
             '0.0.0.0/0')
 
-
-    def format_firewall_rule_name(self, rule_name):
-        """Format firewall rule name.
-
-        Args:
-            rule_name (str): Name of the firewall rule.
-
-        Returns:
-            str: Firewall rule name.
-        """
-        return '{}-{}'.format(rule_name, self.config.identifier)
+    def delete_firewall_rules(self):
+        """Deletes default firewall rules as the forseti service account rules
+        serves the purpose"""
+        if len(self.firewall_rules_to_be_deleted) > 0:
+            for rule in self.firewall_rules_to_be_deleted:
+                gcloud.delete_firewall_rule(rule)
+            print('Firewall rules were successfully deleted')
```

Comment:
I think that we should include what firewall rules where deleted for the users to see.

Reply:
Consider it done.